src/main/java/org/spongepowered/api/data/BoundedProp.java

```diff
@@ -22,11 +22,13 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package org.spongepowered.api.data.manipulators.blocks;
+package org.spongepowered.api.data;
 
-import org.spongepowered.api.data.manipulators.SingleValueData;
-import org.spongepowered.api.data.types.StoneType;
+// TODO Does a numeric property (int, double) imply boundedness for all possible cases?
+public interface BoundedProp<E, V> extends Prop<E, V> {
```

Comment:
In IRC I said that we may want to use some comparisons which aren't the "natural" implementation ,so it will most likely require a getComparator method instead.